test/functional/apps/discover/_data_grid_row_navigation.ts

```diff
@@ -30,8 +30,8 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
 
   describe('discover data grid row navigation', function () {
     before(async () => {
-      await security.testUser.setRoles(['kibana_admin', 'similar-index']);
-      await security.testUser.setRoles(['kibana_admin', 'similar-index-two']);
+      await security.testUser.setRoles(['kibana_admin', 'similar_index']);
+      await security.testUser.setRoles(['kibana_admin', 'similar_index_two']);
```

Comment:
Oh, stuff like this can drive people crazy :) One thing to prevent this would be just use `similar_index` or `similar-index`, would you mind to just choose one of those for rule & index, so this way we would omit a potential future confusion? Everything else LGTM!